src/DMS/Service/Meetup/Response/MultiResultResponse.php

```diff
@@ -42,6 +43,7 @@ public function map(Closure $func)
         $clone = clone $this;
 
         $clone->setData(array_map($func, $this->data));
+        $clone->updateBody();
```

Comment:
@rdohms Since we have a public setter for `data` we should do this operation on that call, but I wanted to check your preference 😉 

I'd rather just remove the setter 😈